content/kubermatic/v2.17/tutorials_howtos/KKP_autoscaler/_index.en.md

```diff
@@ -0,0 +1,238 @@
+
++++
+title = "Kubermatic Kubernetes Platform (KKP) Cluster Autoscaler"
+date = 2021-08-05T14:07:10+02:00
+weight = 21
++++
+
+This section deals with the usage of Kubernetes Cluster Autoscaler in the KKP Cluster. 
+
+## What is a Cluster Autoscaler in Kubernetes?
+
+Kubernetes Cluster Autoscaler is a tool that automatically adjusts the size of the worker’s node up or down depending on the consumption. This means that the Autoscaler, for example, automatically scale up a Cluster by increasing the node size when there are not enough node resources for Cluster workload scheduling and scale down when the node resources have continuously staying idle, or there are more than enough node resources available for Cluster workload scheduling. In a nutshell, it is a component that automatically adjusts the size of a Kubernetes Cluster so that all pods have a place to run and there are no unneeded nodes.
+
+## KKP Cluster Autoscaler Usage
+
+The Kubernetes Autoscaler in the KKP Cluster automatically scaled up/down when one of the following conditions is satisfied:
+
+* Some pods failed to run in the cluster due to insufficient resources.
+* There are nodes in the cluster that have been underutilised for an extended period (10 minutes by default) and can place their Pods on other existing nodes.
+
+ 
+## The Pre-requisites
+
+Using a Kubernetes cluster autoscaler in the KKP cluster must meet specific minimum requirements:
+
+* Kubernetes cluster running Kubernetes v1.18 or newer is required.
+
+
+## Installing Kubernetes Auto-scaler on KKP Cluster
+
+You can install Kubernetes autoscaler on a running KKP Cluster using the KKP addon mechanism, which is already built into the KKP Cluster dashboard.
+
+ 
+
+**Step 1**
+
+Create a KKP Cluster by selecting your project on the dashboard and click on `“create cluster”`. More details can be found on the official [documentation](https://docs.kubermatic.com/kubermatic/master/tutorials_howtos/project_and_cluster_management/) page.  
+
+**Step 2**
+
+When the Cluster is ready, check the Pods in the kube-system Namespace to know if any Autoscaler is running.
+
+![KKP Dashboard](/img/kubermatic/v2.17/tutorials/kkp_autoscaler_dashboard.png?classes=shadow,border "KKP Dashboard")
+
+```bash
+$ kubectl get pods -n kube-system
+
+NAME                              	READY   STATUS     RESTARTS   AGE
+
+canal-gq9gc                       	 2/2    Running       0       21m
+
+canal-tnms8                       	 2/2    Running       0       21m
+
+coredns-666448b887-s8wv8             1/1    Running       0       25m
+
+coredns-666448b887-vldzz             1/1    Running       0       25m
+
+kube-proxy-2whcq                     1/1    Running       0       21m
+
+kube-proxy-tstvd                     1/1    Running       0       21m
+
+node-local-dns-4p8jr                 1/1    Running       0       21m 
```

Comment:
This is more aligned in the `/master/` file.

Reply:
Did you check the output on the preview? Sometimes it might show this way while the output on the main page is well aligned.

Reply:
Ah, I just checked it in the Markdown-Preview of Github, not the in the hugo preview.

Nevermind then. :)